src/Compilers/CSharp/Test/Emit/Emit/IsConstModifierTests.cs

```diff
@@ -0,0 +1,757 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using System.Collections.Immutable;
+using System.Linq;
+using Microsoft.CodeAnalysis.CSharp.Test.Utilities;
+using Xunit;
+using Microsoft.CodeAnalysis.Test.Utilities;
+
+namespace Microsoft.CodeAnalysis.CSharp.UnitTests.Emit
+{
+    public class IsConstModifierTests : CSharpTestBase
```

Comment:
Yes. I filed #19951 and assigned it to myself for that.

---
In reply to: [119788719](https://github.com/dotnet/roslyn/pull/19658#discussion_r119788719) [](ancestors = 119788719,118076845,118076522)